scripts/monitoring/auto_scale_workers.py

```diff
@@ -98,22 +102,65 @@ def get_boto3_client(resource, aws_keys):
 
 
 def increase_or_decrease_workers(challenge):
-    queue_length = get_queue_length_by_challenge(challenge)
-    if queue_length > 0:
-        # start worker
-        start_worker(challenge["id"])
-        print("Started worker for challenge: {}".format(challenge["id"]))
+    try:
+        queue_length = get_queue_length_by_challenge(challenge)
+    except Exception as _e:  # noqa: F841
+        print(
+            "Unable to get the queue length for challenge ID: {}. Skipping.".format(
+                challenge["id"]
+            )
+        )
+        return
+
+    # NOTE: What should be done when workers is None?
+    if challenge["workers"] is None:
```

Comment:
Start the worker when submissions are > 0. `workers` is None when we delete the fargate worker for a challenge